From c3fc6497275f839413c41b0d0c47385363fcb420 Mon Sep 17 00:00:00 2001
From: Nikolay Aleksandrov <nikolay@redhat.com>
Date: Wed, 1 Oct 2014 10:25:48 -0400
Subject: [redhat] RHMAINTAINERS: Update bonding and sky2 maintainer

Message-id: <1412159148-25722-1-git-send-email-nikolay@redhat.com>
Patchwork-id: 96815
O-Subject: [RHEL7 PATCH] RHMAINTAINERS: Update bonding and sky2 maintainer
Bugzilla:
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

Update bonding and sky2 maintainer entries.

Signed-off-by: Nikolay Aleksandrov <nikolay@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/RHMAINTAINERS b/redhat/RHMAINTAINERS
index 2cd17cf..7c76008 100644
--- a/redhat/RHMAINTAINERS
+++ b/redhat/RHMAINTAINERS
@@ -170,7 +170,7 @@ S: Maintained
 F: drivers/net/tg3.*
 
 BONDING DRIVER
-M: Veaceslav Falico <vfalico@redhat.com>
+M: Nikolay Aleksandrov <naleksan@redhat.com>
 S: Maintained
 F: drivers/net/bonding/
 F: include/linux/if_bonding.h
@@ -618,7 +618,7 @@ S: Maintained
 F: drivers/net/skge.*
 
 SKY2 10/100/1000 GIGABIT ETHERNET DRIVER
-M: Andy Gospodarek <agospoda@redhat.com>
+M: Nikolay Aleksandrov <naleksan@redhat.com>
 S: Maintained
 F: drivers/net/sky2.*
 
-- 
1.7.1